src/python/pants/backend/python/register.py

```diff
@@ -90,4 +90,10 @@ def register_goals():
 
 
 def rules():
-  return inject_init.rules() + python_test_runner.rules() + python_native_code_rules() + subprocess_environment_rules()
+  return (
+    inject_init.rules()
+    + python_test_runner.rules()
```

Comment:
Nit: + operator should probably go at the end of the line, so all the operands line up.